appserver/webservices/webservices-scripts/src/main/resources/glassfish/bin/wscompile

```diff
@@ -59,4 +59,11 @@ if [ ${AS_JAVA} ]; then
     JAVA=${AS_JAVA}/bin/java
 fi
 
-exec "$JAVA" $WSCOMPILE_OPTS -Djava.endorsed.dirs="$AS_INSTALL_LIB/endorsed" -cp "$AS_INSTALL_LIB/webservices-osgi.jar:$AS_INSTALL_LIB/javax.xml.rpc-api.jar:$AS_INSTALL_LIB/jaxb-osgi.jar:$JAVAX_MAIL_JAR:$JAVA_HOME/lib/tools.jar:$AS_INSTALL_LIB/aixporting-repackaged.jar" com.sun.xml.rpc.tools.wscompile.Main "$@"
+./jdkcheck
+
+if [ checkEndorsedAvailable = true ]
```

Comment:
I'm not sure this works, either the method is wrong or the if syntax is wrong. Running with the following Java version has it go into the JDK11 block

```
openjdk version "1.8.0_222"
OpenJDK Runtime Environment (IcedTea 3.13.0) (build 1.8.0_222-b10 suse-lp151.2.3.1-x86_64)
OpenJDK 64-Bit Server VM (build 25.222-b10, mixed mode)
```